packages/m/maelstrom/package.yml

```diff
@@ -1,11 +1,13 @@
 name       : maelstrom
 version    : 3.0.7
-release    : 7
+release    : 8
 source     :
     - https://www.libsdl.org/projects/Maelstrom/src/Maelstrom-3.0.7.tar.gz : 93680322f4b04d4727ea8e0b461f259b604dc69e7465cb91d2b9aad378a9a036
+    - https://raw.githubusercontent.com/flathub/org.libsdl.Maelstrom/master/org.libsdl.Maelstrom.appdata.xml : f4ad410e1e12e9517bc5f9af9b73d2dcfdf90a06228d9953afc7d5adc93f498e
```

Comment:
This pulls the file from whatever is currently in the main branch of their repo, which means this isn't a reproducible build; the contents may change unexpectedly. I think I would rather the file be downloaded and put in the `files` directory like the `.desktop` file is, and install it from the pkgfiles.

Reply:
That's right. Why did i pull from master? It should have been from specific commit but putting the file in `$pkgfiles` might be more robust after all. I will change it.